docs/how_it_works/univariate_drift_detection.rst

```diff
@@ -192,6 +222,21 @@ To help our intuition we can look at the image below:
 We see how the relative frequencies of three categories have changed between reference and analysis data.
 We also see that the JS Divergence contribution of each change and the resulting JS distance.
 
+.. _univ_cat_method_hellinger:
+
+Hellinger Distance
+........................
+
+The Hellinger Distance, is a distance metric used to quantify the similarity between two distributions. It measures the overlap between the probabilities assigned 
+to the same event by both reference and analysis. It ranges from 0 to 1 where a value of 1 is only achieved when reference assigns zero probability to each event to which 
+analysis assigns some positive probability and vice versa. 
+The formula is given by:
+
+For a categorical feature Hellinger Distance is defined as:
+
+.. math::
+ H\left(X_i^{ref},X_i^{ana}\right) = \frac{1}{\sqrt{2}}\left[\sum_{x \in X}\left(\sqrt{{F}_{ref}(x)}-\sqrt{{F}_{ana}(x)}\right)^2dx\right]^{1/2}
+
```

Comment:
It would also be good to explain that F in the categorical case correspond to Probability Mass Functions.